206. Reverse Linked List/SecondSolution.java

```diff
@@ -0,0 +1,37 @@
+public class SecondSolution {
+    // Time: O(n), Space: O(n), n は LinkedList の長さ
+    public ListNode reverseList(ListNode head) {
+        return reverseListHelper(null, head);
+    }
+
+    private ListNode reverseListHelper(ListNode prev, ListNode current) {
+        if (current == null) {
+            return prev;
+        }
```

Comment:
NITS: 処理が一行しかない場合、個人的にはこういう書き方が好みです。
```suggestion
        if (current == null) return prev;
```

Reply:
個人的には元の書き方のほうをよく書きます。コーディング規約を確認しつつ、所属するチーム内でメジャーな書き方に合わせられるとよいと思います。

Reply:
ありがとうございます！両方の記法があることを意識しつつ合わせられるようにしていこうとおもいます！

Reply:
Google はぶら下げないんですよね。これは場所によると思います。
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used